flake.nix

```
@@ -70,6 +70,11 @@
       default = webcord;
     });
 
+    homeManagerModules = rec {
+      webcord = import ./hm-module.nix;
+      default = webcord;
```

```suggestion
      default = self.homeManagerModules.webcord;
```
This avoids having to use `rec`.